src/ClickHouseStatement.php

```diff
@@ -292,8 +292,8 @@ public function execute($params = null) : bool
         } else {
             foreach (array_keys($this->values) as $key) {
                 $sql = preg_replace(
-                    '/(' . (is_int($key) ? '\?' : ':' . $key) . ')/i',
-                    $this->getTypedParam($key),
+                    '/(' . (is_int($key) ? '\?([,\)])' : ':' . $key) . ')/i',
```

Comment:
what do you think about this expr: `\?([,\)]|\s,|\s\))` for resolve next case with whitespace character: `'? , ? )` ?

Reply:
I haven't seen this situation, but yes, we can add just spaces
```suggestion
                    '/(' . (is_int($key) ? '\?(\s?[,\)])' : ':' . $key) . ')/i',
```

Reply:
Anyway, it don't help if we will have `?,` in the text :)